doc/string/partition.rdoc

```diff
@@ -0,0 +1,22 @@
+Returns a 3-element array of substrings of +self+.
+
+Matches a pattern against +self+; the pattern is:
```

Comment:
I think the wording of the description should be different for `String#partition` and `String#rpartition` to highlight the difference.

```suggestion
Matches a pattern against +self+ from the beginning, scanning forwards. The pattern is:
```

Reply:
Fixed.